### DISCUSSION TITLE
Host icons locally?

### DISCUSSION THREAD

**Discussion by BadBull86** [matt8707/ha-fusion#441] [Ideas]
I blocked the internet connection of my tablet. Therefore I am missing the icons in my dashboard. Is it possible to host them locally or integrate them in the ha-Fusion container?
Or is it not allowed due to copyright reasons?

**Comment by matt8707** [comment]
I did try that, but with all the icons, it's several gigabytes. I could add mdi icons though, so at least they work offline.

**Reply by BadBull86** [reply]
Thanks for your answer!
I like the icons you are using, so you should keep them. That means to me I will change them to mdi manually or turn on internet..

Or is it easy to integrate a toggle, where the user can choose which set to use?
With a restart for taking change.

**Reply by matt8707** [reply]
> Or is it easy to integrate a toggle, where the user can choose which set to use?

That would still require that fusion includes all the icons. The way it works now are that icons are fetched from online and cached in your browser.

You can read more here https://iconify.design/

**Reply by BadBull86** [reply]
Makes sense.
Do you know the address + port how we connect to the api?
So I can whitelist this one.

**Reply by matt8707** [reply]
looks like it calls cloudflare at https://api.iconify.design which points to 104.26.13.204:443

**Reply by BadBull86** [reply]
Seems to work - thanks!!
